bundles/org.openhab.binding.speedtest/pom.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>3.4.0-SNAPSHOT</version>
```

Comment:
```suggestion
    <version>4.0.0-SNAPSHOT</version>
```

Reply:
done.

in addition I updated the (c) to 2023.

Reply:
@wborn, what's needed in addition in order to get this one finally merged?

Reply:
Thanks for your patience. 🙏 One of the maintainers needs to find some time to review it.  🧑‍💻⏳